Frontend/src/components/TaskList.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import CsrfToken from './CsrfToken';
+
+export interface ITask {
+  id: number;
+  title: string;
+  status: boolean;
+}
+
+export async function getTasks(): Promise<ITask[]> {
+  const resp = await axios.get('list_tasks');
+  return resp.data.tasks;
+}
+
+export default function TaskList() {
+  const [tasks, setTasks] = useState<ITask[]>([]);
+
+  useEffect(() => {
+    const getResp = async () => {
```

Comment:
The name must be more descriptive - the `getResp` doesn't tell us anything about what's going on inside the function. For example: `getTasks` and the other one on line #11 could be `fetchTasks`. I know they are both named in a similar way, but that doesn't mean we should resort to using very general naming like `getResp`

Reply:
Roger that. Renamed to `taskHandler`, I think that might be a little more appropriate.